src/rfpimp.py

```diff
@@ -350,7 +350,7 @@ def dropcol_importances(model, X_train, y_train, X_valid=None, y_valid=None, met
     return I
 
 
-def oob_dropcol_importances(rf, X_train, y_train):
+def oob_dropcol_importances(rf, X_train, y_train, metric=None):
```

Comment:
Will this default to same behavior (i.e., True oob score) for backward compatibility?

Reply:
Yes, I prefered to set the default value to `None` as opposed to True (and then set it to `True` on [L376](https://github.com/ValterH/random-forest-importances/blob/d40d04913338ef4b6bc5ee49d60fe98c3f9eff8f/src/rfpimp.py#L376)) as I think it is more intuitive, and in line with other methods (e.g. [`importances`](https://github.com/ValterH/random-forest-importances/blob/d40d04913338ef4b6bc5ee49d60fe98c3f9eff8f/src/rfpimp.py#L73))

Reply:
oh sorry. missed that.